connectors/agentic-ai/bin/transform-ai-agent-job-worker-template.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
Temporary solution to transform the outbound connector template until implementing https://github.com/camunda/connectors/issues/5189.

- needs `jq` to be available
- is not automatically executed (needs to be run manually)